src/components/file/file.tsx

```diff
@@ -67,6 +68,18 @@ export class File {
     @Prop({ reflect: true })
     public label: string;
 
+    /**
+     * Optional file icon.
+     */
+    @Prop()
+    public fileIcon: string;
+
+    /**
+     * Optional file icon color.
+     */
+    @Prop()
+    public fileIconColor: string;
```

Comment:
Since this sets the background colour of the icon "badge", we will probably want to be able to set the icon colour too, before too long. I'm thinking we should probably add that while we're at it. What do you think?

Reply:
That was my initial thought as well. However I backed on that when I saw that the `chip-set` component doesn't seem to have support for that other than via CSS since the `iconColor` property on the `Chip` type actually sets the _background colour_ which is a bit confusing. So don't know what's the best approach here.

Reply:
Hmm… I wanted to rename this to something like `fileIconBackgroundColor`, but found this in `chip.types.ts`:

![image](https://user-images.githubusercontent.com/154725/102238324-e7b15400-3ef5-11eb-9ada-0296b4c88a75.png)

@jgroth do you remember why we had to override `--icon-background-color` in the first place?

Reply:
Hence the confusion ↑

Reply:
Ah, yeah, I hadn't seen your comment 😆

Reply:
Turns out both colours can be set using the appropriate css variables. So I revert part of your change, and added proper documentation for how to use those variables.

Reply:
I guess that this can be used if you want different colors for the chips if you have multiple ones as part of a chip set. Setting the CSS property will set the same color on all of the chips

Reply:
Ah, yeah, that makes sense 👍

`limel-file` only supports one chip at the moment though, so we can use the css variables in this case 🙂